make_conan_pkg.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+conan source . --source-folder=tmp/source && \
+conan export external/rocksdb/ && \
+conan install . --install-folder=tmp/build && \
+conan build . --source-folder=tmp/source --build-folder=tmp/build && \
+conan package . --source-folder=tmp/source --build-folder=tmp/build --package-folder=tmp/package && \
+conan export-pkg . --package-folder=tmp/package
```

Comment:
Do we really need or want this script? Maybe it can go into `BUILDING.md` as instructions. If you really want to keep this script, maybe put it into `bin/`, add a comment explaining what it does and how it is supposed to be called, and change it from one line to multiple lines but with `set -o errexit` at the top.

Reply:
I'll remove this and just make it build instructions.